src/scripts/App.ts

```diff
@@ -0,0 +1,31 @@
+class App {
+
+    static readonly debug = false;
+    static game: Game;
+
+    static start() {
+        if (!App.debug)
+            Object.freeze(GameConstants);
```

Comment:
I know this works as is, but would be nice to have some { } around the !debug code

Reply:
100% agree, must have missed it while copy-pasting